packages/fast-css-editor-react/src/position/index.tsx

```diff
@@ -0,0 +1,173 @@
+import * as React from "react";
+import { IManagedClasses } from "@microsoft/fast-components-class-name-contracts-base";
+import manageJss, { IJSSManagerProps } from "@microsoft/fast-jss-manager-react";
+import style, { ICSSPositionClassNameContract } from "./position.style";
+
+export enum PositionValue {
+    static = "static",
+    absolute = "absolute"
+}
+
+export enum Location {
+    top = "top",
+    left = "left",
+    right = "right",
+    bottom = "bottom"
+}
+
+export interface ILocationsMappedToClassNames {
+    location: Location;
+    className: string;
+}
+
+export interface ICSSPositionProps {
+    position?: PositionValue;
+    left?: number;
+    right?: number;
+    top?: number;
+    bottom?: number;
+    onChange: (positionValues: any) => void;
+}
+
+class CSSPosition extends React.Component<ICSSPositionProps & IManagedClasses<ICSSPositionClassNameContract>, {}> {
+    private positionKey: string = "position";
+
+    public render(): JSX.Element {
+        return (
+            <div className={this.props.managedClasses.cssPosition}>
+                <span className={this.props.managedClasses.cssPosition_selectContainer}>
```

Comment:
Not a blocker, but <span> here seems better served as a <div>. I could see using the <span> if we want it to function as an inline-element here, but since we are applying styles to it, it seems out of place. 

Total nit. Not blocking. More curious than anything.

Reply:
Not sure I understand your comment?

Reply:
Just curious on the choice of wrapping select in a span here if it's not inline content